docs/getting-started.md

```diff
@@ -167,15 +168,16 @@ are using, if you are using something else:
 make publish-component-containers CONTAINER_REPO=localhost:5000/ocurity/dracon
 ```
 
-\* Notice that the repo we are using is slightly different than the
+*Notice that the repo we are using is slightly different from the
 one we pushed the images in the previous step. That's because with local
 registries the registry is exposed on a port in localhost, however inside the
-KiND cluster, that's not the case. Instead the registry's host is
+KiND cluster, that's not the case. Instead, the registry's host is
 `kind-registry:5000`. This is also going to be important later when we will
 deploy the pipelines and their image repositories will also have to be set to
 this value.
-
-\*\*Make sure that you use the draconctl image that you pushed in the repository
```

Comment:
this change doesn't look correct. the idea is that you have a first footnote (\*) and then a second one to check (\*\*)

Reply:
Fixed, it now looks like this

![Screenshot 2024-09-05 at 11 50 03](https://github.com/user-attachments/assets/6110b3d7-b035-4cb3-b55a-4cc004e4e611)